src/.expo/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+  "hostType": "lan",
+  "lanType": "ip",
+  "dev": true,
+  "minify": false,
+  "urlRandomness": null
```

Comment:
Is this change and the "packager-info.json" necessary @cykler?

Reply:
Don't know where this came from but I removed it for now.